diff -Nuar src/FlPosteRazorDialog.cpp src-fixed/FlPosteRazorDialog.cpp
--- src/FlPosteRazorDialog.cpp	2008-07-21 22:24:40.000000000 +0000
+++ src-fixed/FlPosteRazorDialog.cpp	2017-05-12 11:43:07.704479215 +0000
@@ -432,7 +432,7 @@
 		loaded = m_posteRazorController->loadInputImage(loadFileName, errorMessage, sizeof(errorMessage));
 		if (!loaded) {
 			if (strlen(errorMessage) > 0)
-				fl_message(errorMessage);
+				fl_message("%s", errorMessage);
 			else
 				fl_message(Translations::instance().fileCouldNotBeLoaded(), fl_filename_name(loadFileName));
 		}
@@ -603,7 +603,7 @@
 			fileExistsAskUserForOverwrite = my_file_exists(saveFileName);
 			char overwriteQuestion[1024] = "";
 			sprintf(overwriteQuestion, Translations::instance().overwriteFile(), fl_filename_name(saveFileName));
-			if (!fileExistsAskUserForOverwrite || fl_ask(overwriteQuestion)) {
+			if (!fileExistsAskUserForOverwrite || fl_ask("%s", overwriteQuestion)) {
 				int err = m_posteRazorController->savePoster(saveFileName);
 				if (err)
 					fl_message(Translations::instance().fileCouldNotBeSaved(), fl_filename_name(saveFileName));
diff -Nuar src/FlPosteRazorHelpDialog.cpp src-fixed/FlPosteRazorHelpDialog.cpp
--- src/FlPosteRazorHelpDialog.cpp	2008-07-21 22:24:40.000000000 +0000
+++ src-fixed/FlPosteRazorHelpDialog.cpp	2017-05-12 11:48:26.484298706 +0000
@@ -22,6 +22,8 @@
 
 #include "FlPosteRazorHelpDialog.h"
 #include "Translations.h"
+#include <strings.h>
+#include <string.h>
 
 #if defined (WIN32)
   #include <Shellapi.h>
diff -Nuar src/FlPosteRazorSpinner.cpp src-fixed/FlPosteRazorSpinner.cpp
--- src/FlPosteRazorSpinner.cpp	2008-07-21 22:24:40.000000000 +0000
+++ src-fixed/FlPosteRazorSpinner.cpp	2017-05-12 11:49:15.812971929 +0000
@@ -23,6 +23,7 @@
 #include "FlPosteRazorSpinner.h"
 #include <FL/Fl_Input.H>
 #include <FL/Fl_Repeat_Button.H>
+#include <stdlib.h>
 #include <stdio.h>
 #include <math.h>
 #include <cstdlib>
diff -Nuar src/PosteRazorPDFOutput.cpp src-fixed/PosteRazorPDFOutput.cpp
--- src/PosteRazorPDFOutput.cpp	2008-07-21 22:24:40.000000000 +0000
+++ src-fixed/PosteRazorPDFOutput.cpp	2017-05-12 11:44:37.103518140 +0000
@@ -397,7 +397,7 @@
 		int err = 0;
 
 		const unsigned int startxref = ftell(m_outputFile);
-		fprintf(m_outputFile, m_xref);
+		fprintf(m_outputFile, "%s", m_xref);
 
 		fprintf (
 			m_outputFile,
